src/dc_mimeparser.rs

```diff
@@ -995,6 +995,19 @@ impl<'a> MimeParser<'a> {
 
         assert_eq!(self.parts.len(), 1);
     }
+
+    pub fn get_rfc724_mid(&mut self) -> Option<String> {
+        // get Message-ID from header
+        if let Some(field) = self.lookup_field_typ("Message-ID", MAILIMF_FIELD_MESSAGE_ID) {
+            unsafe {
+                let fld_message_id = (*field).fld_data.fld_message_id;
+                if !fld_message_id.is_null() {
+                    return Some(as_str((*fld_message_id).mid_value).to_string());
```

Comment:
should be `to_string(..)` instead of `as_str(..).to_string()` (I think this happens in some other places too)

Reply:
fixed it -- but found no other instances of `as_str.*to_string()`